src/components/Input/styles/InputStyles.js

```diff
@@ -7,7 +7,7 @@ const InputStyles = styled.input`
   font-size: 14px;
   font-family: ${get('fonts.brand')};
   transition: all 0.2s ease;
-  padding: ${get('spacing.medium')};
+  padding: calc(${get('spacing.medium')} - ${get('thicknesses.normal')});
```

Comment:
This seems a bit arbitrary, since the border thickness is `thicknesses.wide`.

Reply:
What I mean is, I'd be happier with just `1px`. I think using a named value implies more specific meaning, which can be confusing.